arangod/Auth/UserManager.cpp

```diff
@@ -414,7 +414,8 @@ void auth::UserManager::triggerGlobalReload() {
   while (maxTries-- > 0) {
     AgencyCommResult result = agency.sendTransactionWithFailover(incrementVersion);
     if (result.successful()) {
-      _globalVersion.fetch_add(1, std::memory_order_release);
+      uint64_t aha = _globalVersion.fetch_add(1, std::memory_order_release);
+      LOG_DEVEL << "INCREMENTED GLOBAL VERSION TO " << (aha + 1);
```

Comment:
WTF?

Reply:
Oops. Of course this was not supposed to be in this commit :smile: